test/encoding-length.test.js

```diff
@@ -0,0 +1,50 @@
+var fs = require('fs')
+var path = require('path')
+var test = require('tape').test
+var bencode = require('..')
+
+var torrent = fs.readFileSync(
+  path.join(__dirname, '..', 'benchmark', 'test.torrent')
+)
+
+test('encoding-length', function (t) {
```

Comment:
could you maybe add a few simple test cases for non-empty lists/dicts? 
I know the `torrent` test case technically covers that, but I think it would be nice to see those in isolation to prevent a  case where multiple errors could cancel them out in a specific test file.

Reply:
Done!